infra/modules/vpc/main.tf

```
@@ -0,0 +1,45 @@
+locals {
+  project_name = var.project_name
+  vpc_name     = "vpc-${local.project_name}"
```

The vpc name could probably just be the project name
```suggestion
  vpc_name     = local.project_name
```